packages/react-ui-ag/src/Filters/FilterCard.js

```diff
@@ -0,0 +1,71 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { Card, Title, Text, Button } from '@rentpath/react-ui-core'
+
+@themed('*', {
+  pure: true,
+})
+
+export default class FilterCard extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    children: PropTypes.node,
+    title: PropTypes.string,
+    description: PropTypes.string,
+    handleApplyClick: PropTypes.func,
+    handleCancelClick: PropTypes.func,
+  }
+
+  static defaultProps = {
+    theme: {},
+    title: '',
```

Comment:
Do we even need `title` and `description` defaults?

Reply:
I guess not.